components/Git/ArticleEditor.tsx

```diff
@@ -39,6 +35,7 @@ export type HyperLink = HTMLAnchorElement | HTMLImageElement;
 export class ArticleEditor extends Component {
   @observable
   accessor repository = '';
+  accessor editorContent = '';
```

Comment:
```suggestion

  editorContent = '';
```
没有什么特殊需要用普通属性就好了，不需要存取器属性。